### ISSUE TITLE
Bug: Imported Trello board activities are dated to time of import

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#1157]
From @soerenk at #1150

> I used https://github.com/wekan/wekan/wiki/virtual-appliance for testing. Import of my Trello Board is working. Members can be mapped. Labels, Lists, Cards everything was there. Minior problem is, that all activities was dated at the time of the import. That lead to an unsorted Comment section.

@GhassenRjab 

Can you look at this? Can date and time of activities be imported too?

**Comment by xet7** [comment]
I have not tested does this also happen with Import / Export of Wekan board.

**Comment by soerenk** [comment]
I have created an new Weka board on the same VM. Waited a couple of minutes and reimported it to a new board. Looks like it handels the import as a new activity and overrides the created date with the current time of importation. But I am no expert, may look yourself:

### Export
`{"_format":"wekan-board-1.0.0","_id":"ywSHyCF2E7oWBqxk9","title":"Test Wekan","permission":"private","slug":"test-wekan","archived":false,"createdAt":"2017-08-03T10:45:07.666Z","labels":[{"color":"green","_id":"d3arsK","name":""},{"color":"yellow","_id":"akwjkZ","name":""},{"color":"orange","_id":"FJg52b","name":""},{"color":"red","_id":"uNcyiy","name":""},{"color":"purple","_id":"gx5wBd","name":""},{"color":"blue","_id":"B2qAWr","name":""}],"members":[{"userId":"mZkeut9qWN5yqzcM5","isAdmin":true,"isActive":true,"isCommentOnly":false}],"color":"belize","lists":[{"_id":"feKgtSXumecj4d9jw","title":"Liste 1","sort":1,"archived":false,"createdAt":"2017-08-03T10:45:16.970Z"},{"_id":"xg94BSFTy6L3WKJ9S","title":"Liste 1+1","sort":2,"archived":false,"createdAt":"2017-08-03T10:45:53.302Z"},{"_id":"ywcTj6oG5PbdfZBvx","title":"Liste 1+n","sort":3,"archived":false,"createdAt":"2017-08-03T10:46:02.101Z"}],"activities":[{"_id":"rnDQ64md9oyfAwR3G","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklistItem","cardId":"DYbsKLy6DZbF87yrS","checklistId":"ubhyMmnouEQh4E3Fi","checklistItemId":"ubhyMmnouEQh4E3Fi1","createdAt":"2017-08-03T10:46:36.047Z"},{"_id":"dvhMFx3tgZwSsQNQ6","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklistItem","cardId":"DYbsKLy6DZbF87yrS","checklistId":"ubhyMmnouEQh4E3Fi","checklistItemId":"ubhyMmnouEQh4E3Fi0","createdAt":"2017-08-03T10:46:34.527Z"},{"_id":"hGamD4wWxhqnxRYN3","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklist","cardId":"DYbsKLy6DZbF87yrS","checklistId":"ubhyMmnouEQh4E3Fi","createdAt":"2017-08-03T10:46:32.545Z"},{"_id":"etgRyWQNbaBfuG7mZ","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklistItem","cardId":"DYbsKLy6DZbF87yrS","checklistId":"pt8pFKHv2YbptFkFK","checklistItemId":"pt8pFKHv2YbptFkFK1","createdAt":"2017-08-03T10:46:28.616Z"},{"_id":"wNvFTXgm5bu2BqSER","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklistItem","cardId":"DYbsKLy6DZbF87yrS","checklistId":"pt8pFKHv2YbptFkFK","checklistItemId":"pt8pFKHv2YbptFkFK0","createdAt":"2017-08-03T10:46:25.832Z"},{"_id":"B3Ny9YB4SZ5RgtPkG","userId":"mZkeut9qWN5yqzcM5","activityType":"addChecklist","cardId":"DYbsKLy6DZbF87yrS","checklistId":"pt8pFKHv2YbptFkFK","createdAt":"2017-08-03T10:46:22.482Z"},{"_id":"NEWqHqZgmPEr7CW67","userId":"mZkeut9qWN5yqzcM5","activityType":"createCard","listId":"ywcTj6oG5PbdfZBvx","cardId":"DYbsKLy6DZbF87yrS","createdAt":"2017-08-03T10:46:16.338Z"},{"_id":"dKJLiZcbfoRwhWxE2","userId":"mZkeut9qWN5yqzcM5","activityType":"createCard","listId":"xg94BSFTy6L3WKJ9S","cardId":"4EHvSvHhCNkZ9qvfx","createdAt":"2017-08-03T10:46:10.125Z"},{"_id":"dx38EvF7yqDxHQ6PZ","userId":"mZkeut9qWN5yqzcM5","type":"list","activityType":"createList","listId":"ywcTj6oG5PbdfZBvx","createdAt":"2017-08-03T10:46:02.107Z"},{"_id":"M2izvQ4Ro8DWuiKwR","userId":"mZkeut9qWN5yqzcM5","type":"list","activityType":"createList","listId":"xg94BSFTy6L3WKJ9S","createdAt":"2017-08-03T10:45:53.309Z"},{"_id":"YJFHy9RmPH9E7odDo","userId":"mZkeut9qWN5yqzcM5","activityType":"createCard","listId":"feKgtSXumecj4d9jw","cardId":"5CgWLgmuiT6cNTjfN","createdAt":"2017-08-03T10:45:28.390Z"},{"_id":"p8u3hfJbBboh2PKxT","userId":"mZkeut9qWN5yqzcM5","type":"list","activityType":"createList","listId":"feKgtSXumecj4d9jw","createdAt":"2017-08-03T10:45:16.979Z"},{"_id":"rdhMTmtWxeDo89KF8","userId":"mZkeut9qWN5yqzcM5","type":"board","activityTypeId":"ywSHyCF2E7oWBqxk9","activityType":"createBoard","createdAt":"2017-08-03T10:45:07.712Z"}],"cards":[{"_id":"DYbsKLy6DZbF87yrS","title":"Checkliste 1","members":[],"labelIds":["d3arsK"],"listId":"ywcTj6oG5PbdfZBvx","sort":0,"archived":false,"createdAt":"2017-08-03T10:46:16.320Z","dateLastActivity":"2017-08-03T10:46:40.491Z","userId":"mZkeut9qWN5yqzcM5"},{"_id":"4EHvSvHhCNkZ9qvfx","title":"Karte 2","members":[],"labelIds":["uNcyiy"],"listId":"xg94BSFTy6L3WKJ9S","sort":0,"archived":false,"createdAt":"2017-08-03T10:46:10.118Z","dateLastActivity":"2017-08-03T10:46:45.587Z","userId":"mZkeut9qWN5yqzcM5"},{"_id":"5CgWLgmuiT6cNTjfN","title":"Karte 1","members":[],"labelIds":["gx5wBd"],"listId":"feKgtSXumecj4d9jw","sort":0,"archived":false,"createdAt":"2017-08-03T10:45:28.382Z","dateLastActivity":"2017-08-03T10:46:48.857Z","userId":"mZkeut9qWN5yqzcM5"}],"comments":[],"checklists":[{"_id":"pt8pFKHv2YbptFkFK","cardId":"DYbsKLy6DZbF87yrS","title":"ITEM","createdAt":"2017-08-03T10:46:22.443Z","items":[{"_id":"pt8pFKHv2YbptFkFK0","title":"LOL","isFinished":false},{"_id":"pt8pFKHv2YbptFkFK1","title":"ROFL","isFinished":false}],"userId":"mZkeut9qWN5yqzcM5"},{"_id":"ubhyMmnouEQh4E3Fi","cardId":"DYbsKLy6DZbF87yrS","title":"HUHU","createdAt":"2017-08-03T10:46:32.518Z","items":[{"_id":"ubhyMmnouEQh4E3Fi0","title":"HI","isFinished":false},{"_id":"ubhyMmnouEQh4E3Fi1","title":"MOIN","isFinished":false}],"userId":"mZkeut9qWN5yqzcM5"}],"attachments":[],"users":[{"_id":"mZkeut9qWN5yqzcM5","username":"soeren","profile":{}}]}`

### Import:
`{"_format":"wekan-board-1.0.0","_id":"v3QR88FMT2v6KR95Z","archived":false,"color":"belize","createdAt":"2017-08-03T10:50:18.195Z","labels":[{"_id":"Q8stYb","color":"green"},{"_id":"ALXr2X","color":"yellow"},{"_id":"QkfyNR","color":"orange"},{"_id":"axTxsg","color":"red"},{"_id":"yQuxuw","color":"purple"},{"_id":"T92ZwB","color":"blue"}],"members":[{"userId":"mZkeut9qWN5yqzcM5","isAdmin":true,"isActive":true,"isCommentOnly":false}],"permission":"private","slug":"test-wekan","title":"Test Wekan","modifiedAt":"2017-08-03T10:50:18.203Z","lists":[{"_id":"aWFhysuK6KBWCcnwi","archived":false,"createdAt":"2017-08-03T10:50:18.209Z","title":"Liste 1","updatedAt":"2017-08-03T10:50:18.213Z"},{"_id":"iNbe5z4iPx9kd5okE","archived":false,"createdAt":"2017-08-03T10:50:18.218Z","title":"Liste 1+1","updatedAt":"2017-08-03T10:50:18.221Z"},{"_id":"TzPi4mESBdSeLqEzt","archived":false,"createdAt":"2017-08-03T10:50:18.223Z","title":"Liste 1+n","updatedAt":"2017-08-03T10:50:18.224Z"}],"activities":[{"_id":"LJJQjM7hEhKPBP4PN","activityType":"importBoard","createdAt":"2017-08-03T10:50:18.159Z","source":{"id":null,"system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"vkvnfpQ7nCtexwvea","activityType":"importList","createdAt":"2017-08-03T10:50:18.159Z","listId":"aWFhysuK6KBWCcnwi","source":{"id":"feKgtSXumecj4d9jw","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"vyBSEcHZzvteN8Ess","activityType":"importList","createdAt":"2017-08-03T10:50:18.159Z","listId":"iNbe5z4iPx9kd5okE","source":{"id":"xg94BSFTy6L3WKJ9S","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"MPC9xk3yHFCMhjFnv","activityType":"importList","createdAt":"2017-08-03T10:50:18.159Z","listId":"TzPi4mESBdSeLqEzt","source":{"id":"ywcTj6oG5PbdfZBvx","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"F7YByQGH5S6pRyajJ","activityType":"importCard","cardId":"Y22B2yq4HHYYuiBEz","createdAt":"2017-08-03T10:50:18.159Z","listId":"TzPi4mESBdSeLqEzt","source":{"id":"DYbsKLy6DZbF87yrS","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"qdRiSbxCe2TEPMXgB","activityType":"importCard","cardId":"b6hwdNDED5DfMXh96","createdAt":"2017-08-03T10:50:18.159Z","listId":"iNbe5z4iPx9kd5okE","source":{"id":"4EHvSvHhCNkZ9qvfx","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"},{"_id":"4XwhptuikvDfXp7YB","activityType":"importCard","cardId":"3DxJMpgemmeHTP8n7","createdAt":"2017-08-03T10:50:18.159Z","listId":"aWFhysuK6KBWCcnwi","source":{"id":"5CgWLgmuiT6cNTjfN","system":"Wekan"},"userId":"mZkeut9qWN5yqzcM5"}],"cards":[{"_id":"3DxJMpgemmeHTP8n7","archived":false,"createdAt":"2017-08-03T10:50:18.239Z","dateLastActivity":"2017-08-03T10:50:18.239Z","listId":"aWFhysuK6KBWCcnwi","sort":0,"title":"Karte 1","userId":"mZkeut9qWN5yqzcM5","dueAt":null,"labelIds":["yQuxuw"]},{"_id":"b6hwdNDED5DfMXh96","archived":false,"createdAt":"2017-08-03T10:50:18.235Z","dateLastActivity":"2017-08-03T10:50:18.235Z","listId":"iNbe5z4iPx9kd5okE","sort":0,"title":"Karte 2","userId":"mZkeut9qWN5yqzcM5","dueAt":null,"labelIds":["axTxsg"]},{"_id":"Y22B2yq4HHYYuiBEz","archived":false,"createdAt":"2017-08-03T10:50:18.228Z","dateLastActivity":"2017-08-03T10:50:18.228Z","listId":"TzPi4mESBdSeLqEzt","sort":0,"title":"Checkliste 1","userId":"mZkeut9qWN5yqzcM5","dueAt":null,"labelIds":["Q8stYb"]}],"comments":[],"checklists":[{"_id":"Zx3KnF3WWtyMDDTjN","cardId":"Y22B2yq4HHYYuiBEz","title":"ITEM","createdAt":"2017-08-03T10:50:18.243Z","items":[{"_id":"Zx3KnF3WWtyMDDTjN0","title":"LOL","isFinished":false},{"_id":"Zx3KnF3WWtyMDDTjN1","title":"ROFL","isFinished":false}]},{"_id":"fy5kJFRyzwTshS9BF","cardId":"Y22B2yq4HHYYuiBEz","title":"HUHU","createdAt":"2017-08-03T10:50:18.265Z","items":[{"_id":"fy5kJFRyzwTshS9BF0","title":"HI","isFinished":false},{"_id":"fy5kJFRyzwTshS9BF1","title":"MOIN","isFinished":false}]}],"attachments":[],"users":[{"_id":"mZkeut9qWN5yqzcM5","username":"soeren","profile":{}}]}`

**Comment by GhassenRjab** [comment]
I noticed the issue before but I thought it was the desirable behavior when importing boards.
It will be fixed soon

**Comment by GhassenRjab** [comment]
When importing boards, we used to create new activities about importing the board, lists, cards, etc...
and ignore the original board's activities.
What I am doing right now is import the original board's activities and create a new activity about importing the board. Unfortunately I encountered a problem causing attachments' related activities to be created twice. I reported the [issue](https://github.com/matb33/meteor-collection-hooks/issues/225) causing the problem to the library we are using to store attachments. Once I have the solution, I'll create the PR fixing this issue.
